haystack/templates/indexing.yaml.jinja2

```
@@ -12,7 +12,7 @@ components:
   embedder: {{ embedder | default({
     "init_parameters": {
       "batch_size": 32,
-      "device": "cpu",
+      "device": {'type': 'single', 'device': 'cpu'},
```

Let's keep it consistent and use double quotes.
```suggestion
      "device": {"type": "single", "device": "cpu"},
```